### ISSUE TITLE
Manage text-align for docx text styling 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#243]
```
Manage text-align for docx text styling (ex :with HTML use 
style="text-align:..."
```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 6 Apr 2013 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
This feature is available for 1.0.1. See Git commit at 
https://code.google.com/p/xdocreport/source/detail?r=a8d871208621e09b1254542385c
1e0d31e1bcb99
```

Original comment by `angelo.z...@gmail.com` on 6 Apr 2013 at 1:25
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
cleanup
```

Original comment by `pascal.leclercq` on 10 Oct 2014 at 7:37
